checkpoint3/Program.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Sqlite;
+using System.Linq;
+
+namespace checkpoint3
+{
+    public class App
+    {
+        static void Main(string[] args)
+        {
+            App app = new App();
+            app.start();
+            
+        }
+        public Dao dao;
+        public UI ui;
+        public void start()
+        {
+            ui.PrintMenu();
+            ui.GetInput();
+
+        }
+        public App()
+        {
+            Dao dao = new Dao();
```

Comment:
You should not define a new dao variable, but use the field for the class. So it should be:
```dao = new Dao();``` instead of ``` Dao dao = new Dao()```